gemrb/core/DisplayMessage.cpp

```diff
@@ -187,30 +188,63 @@ void DisplayMessage::DisplayString(String text, const Color &color, Scriptable *
 	}
 }
 
+std::vector<std::string> DisplayMessage::GetAllColors() const
+{
+	std::vector<std::string> auxiliaryColors;
```

Comment:
Arent those names static? A view type like `std::string_view` would be much better here

Reply:
I don't see any benefit, but there's no reason for the function to work with a temporary either.